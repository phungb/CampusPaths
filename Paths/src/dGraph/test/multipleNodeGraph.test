# Tests adding multiple nodes to Graph

# Create a graph
CreateGraph multiple_node_graph

# Add three nodes
AddNode multiple_node_graph n1
AddNode multiple_node_graph n2
AddNode multiple_node_graph n3

# Print the nodes in the graph
ListNodes multiple_node_graph
