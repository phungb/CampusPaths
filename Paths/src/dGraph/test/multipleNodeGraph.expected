# Tests adding multiple nodes to Graph

# Create a graph
created graph multiple_node_graph

# Add three nodes
added node n1 to multiple_node_graph
added node n2 to multiple_node_graph
added node n3 to multiple_node_graph

# Print the nodes in the graph
multiple_node_graph contains: n1 n2 n3